CLI Compilation Issue

## Bug Description
Encountered a compilation error after trying to compile client-cli
## Code to reproduce
Compile client-cli
A am running Windows 11 and my version of cargo is 1.90.0-nightly

## Additional information
Here is the exact error message that I got:

```
error[E0422]: cannot find struct, variant or union type 'BatteryData' in this scope
   --> packages\client-cli\src\main.rs:137:35
    |
137 |                         .set_data(BatteryData {
    |                                   ^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function 'sleep' in this scope
   --> packages\client-cli\src\main.rs:153:21
    |
153 |                     sleep(Duration::from_millis(20)).await;
    |                     ^^^^^ not found in this scope
    |
help: consider importing one of these functions
    |
1   + use std::thread::sleep;
    |
1   + use tokio::time::sleep;
    |

error[E0433]: failed to resolve: use of undeclared type'Duration'
   --> packages\client-cli\src\main.rs:153:27
    |
153 |                     sleep(Duration::from_millis(20)).await;
    |                           ^^^^^^^^ use of undeclared type 'Duration'
    |
help: consider importing one of these structs
    |
1   + use std::time::Duration;
    |
1   + use tokio::time::Duration;
    |

error[E0061]: this method takes 1 argument but 0 arguments were supplied
   --> packages\client-cli\src\main.rs:149:32
    |
149 |                     if battery.refresh().is_err() {
    |                                ^^^^^^^-- argument #1 of type 'battery::platform::windows::ffi::DeviceHandle' is missing
    |
note: method defined here
   --> C:\Users\ssark\.cargo\registry\src\index.crates.io-1949cf8c6b5b557f\battery-0.7.8\src\platform\windows\device.rs:61:12
    |
61  |     pub fn refresh(&mut self, mut handle: DeviceHandle) -> Result<()> {
    |            ^^^^^^^
help: provide the argument
    |
149 |                     if battery.refresh(/* battery::platform::windows::ffi::DeviceHandle */).is_err() {
    |                                        +++++++++++++++++++++++++++++++++++++++++++++++++++

Some errors have detailed explanations: E0061, E0422, E0425, E0433.
For more information about an error, try 'rustc --explain E0061'.
error: could not compile 'client-cli' (bin "client-cli") due to 4 previous errors
```

--- a/packages/client-cli/src/main.rs
+++ b/packages/client-cli/src/main.rs
@@ -1,4 +1,4 @@
-use std::{option::Option, path::PathBuf, pin::Pin};
+use std::{option::Option, path::PathBuf, pin::Pin, time::Duration};
 
 #[cfg(any(
     target_os = "linux",
@@ -20,6 +20,7 @@ use tokio::{
     net::TcpListener,
     process::Command,
     sync::Mutex,
+    time::sleep,
 };
 use vex_v5_qemu_host::{
     brain::{Binary, Brain},

